.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+on: [push]
+jobs:
+  run-unit-test:
+    runs-on: ubuntu-latest
```

Comment:
Do we want to ping the ubuntu to a specific version?

Reply:
Sure, pinned to ubuntu-20.04